app/tests/components/Form/ApplicationForm.test.tsx

```diff
@@ -166,41 +254,25 @@ describe('The application form', () => {
   });
 
   it('submission page submit button is enabled on when all inputs filled', async () => {
-    componentTestingHelper.loadQuery();
+    componentTestingHelper.loadQuery(submissionPayload);
```

Comment:
`submissionPayload` doesn't have the `submissionCompletedFor` and `submissionDate`, should they be present for the button to be enabled?